packages/contracts/test/mocha/utils.spec.ts

```diff
@@ -1,35 +0,0 @@
-import child_process from 'child_process'
-
-import { expect } from 'chai'
-import sinon from 'sinon'
-
-import { getCurrentGitCommitHash } from '../../src'
-
-describe('getCurrentGitCommitHash', () => {
```

Comment:
These tests have been moved to the plugins package along with the `getCurrentGitCommitHash` function.